docs/config.md

```diff
@@ -8,6 +8,10 @@ Within this directory, there will be a subdirectory for each monthly scan, named
 
 When scaffold is run from the command line, it will load in `config.json`; process the requested actions; and output a revised `config.json` file with the updated status details. It will also store a backup copy of the prior `config.json` file in the `backup/` subdirectory for that month, in case there are any errors that result in an invalid `config.json` file being outputted.
 
+In addition to the `config.json` file, a `finding-[project].yaml` file is needed to generate the HTML report.  Details on the file format are below.
```

Comment:
Believe this should be `findings-` plural, not `finding`?

Reply:
Good catch - thanks @swinslow - fixed in latest commit